### DISCUSSION TITLE
How to deploy app router nextjs on cpanel?

### DISCUSSION THREAD

**Discussion by Danztee** [vercel/next.js#67887] [App Router]
I have a Next.js app where I'm calling some APIs within the page component using top level await and the page component being async, and I want to deploy it on Cpanel. Is it possible?

**Comment by Juneezee** [comment]
It is possible, but you will need to do some extra configurations.

Related links:
1. https://github.com/vercel/next.js/discussions/55411
2. https://www.youtube.com/watch?v=ruuVPm8jveM
